luna/featurevis/transformations.py

```diff
@@ -118,6 +257,7 @@ def gaussian_blur(img, kernel_size=3, sigma=5):
     Returns:
         The altered image.
     """
+
```

Comment:
I would put this function outside. We also did not put the `angle_conversion` into `rotation`.

Reply:
Done